src/common/legacy_config_opts.h

```diff
@@ -1562,3 +1562,6 @@ OPTION(debug_allow_any_pool_priority, OPT_BOOL)
 OPTION(rgw_gc_max_deferred_entries_size, OPT_U64) // GC deferred entries size in queue head
 OPTION(rgw_gc_max_queue_size, OPT_U64) // GC max queue size
 OPTION(rgw_gc_max_deferred, OPT_U64) // GC max number of deferred entries
+
```

Comment:
i'd suggest avoid use cherry-pick if the original change was not included in ceph repo.

Reply:
Done
I dropped the cherry-pick